MAUI/DataGrid/selection.md

```diff
@@ -57,6 +99,26 @@ The `SfDataGrid` provides [SelectedIndex](https://help.syncfusion.com/cr/maui/Sy
 
 `SfDataGrid.SelectedRows`: Provides all the selected records of the selected items when multiple selection is enabled.
 
+### Getting selected cells.
+
+The `SfDataGrid` provides[`SfDataGrid.GetSelectedCells()`] and [`SfDataGrid.GetSelectedCellInfo()`].
```

Comment:
The `SfDataGrid` provides [SfDataGrid.GetSelectedCells()]() and [SfDataGrid.GetSelectedCellInfo()]() methods to get details of the selected cells when the selection unit is `Cell` or `Any`.

Reply:
Addressed the changes